politeiawww/legacy/comments/process.go

```diff
@@ -95,6 +95,70 @@ func (c *Comments) processNew(ctx context.Context, n v1.New, u user.User) (*v1.N
 	}, nil
 }
 
+func (c *Comments) processEdit(ctx context.Context, e v1.Edit, u user.User) (*v1.EditReply, error) {
+	log.Tracef("processEdit: %v %v", e.Token, e.CommentID)
+
+	// Verify state
+	state := convertStateToPlugin(e.State)
+	if state == comments.RecordStateInvalid {
+		return nil, v1.UserErrorReply{
+			ErrorCode: v1.ErrorCodeRecordStateInvalid,
+		}
+	}
+
+	// Verify user signed using active identity
+	if u.PublicKey() != e.PublicKey {
+		return nil, v1.UserErrorReply{
+			ErrorCode:    v1.ErrorCodePublicKeyInvalid,
+			ErrorContext: "not active identity",
+		}
+	}
+
+	// Ensure that session user is the comment author
```

Comment:
This is ambigous.  The comment author implies that this is the original comment author, which is not what this is checking.  You need to be more specific here.  Incorrect documentation is the same as a bug.